modules/mobile/app/models/mobile/v0/adapters/lighthouse_individual_claims.rb

```diff
@@ -108,8 +108,10 @@ def events_timeline(attributes)
           # Add documents not associated with a tracked item
           events += create_events_for_documents(attributes)
 
-          # Make reverse chron with nil date items at the end
-          events.compact.sort_by { |h| h[:date] || Date.new }.reverse
+          # sort to put events with uploaded == false on top and then by date
+          events.compact.sort_by do |event|
+            [event[:uploaded] || !event.key?(:uploaded) ? 0 : 1, event[:date] || Date.new]
+          end.reverse
```

Comment:
Let me know if there's a better way of doing this. It got complicated though because not all events have the field `uploaded` and some have nil dates.

Reply:
I can't think of a better way, but I do think I would prefer it in an actual if structure just to make it a bit more readable

Reply:
cleaned it up some